client/src/components/QandA/AddAnswer.jsx

```diff
@@ -8,67 +9,74 @@ class AddAnswer extends React.Component {
       answerEntry: '',
       nicknameEntry: '',
       emailEntry: '',
-      addedPhotos: []
+      addedPhotos: [],
+      currentQuestionID: props.question_id
     };
-    this.handleChange = this.handleChange.bind(this);
-    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleAnswerInputChange = this.handleAnswerInputChange.bind(this);
+    this.handleSubmitAnswer = this.handleSubmitAnswer.bind(this);
   }
 
-  handleChange(e) { // this function isn't yet implemented. FIX ME
-    if (e.target === 'answer') { // determine distinguishing target value for corresponding input
+  handleSubmitAnswer(e, currentQuestionID) {
+    e.preventDefault();
+    const { answerEntry, nicknameEntry, emailEntry } = this.state;
+    axios({
+      url: `/qa/questions/${currentQuestionID}/answers`,
+      method: 'post',
+      data: {
+        body: answerEntry,
+        name: nicknameEntry,
+        email: emailEntry,
+        photos: []
+      }
+    });
+    this.setState({ answerEntry: '', nicknameEntry: '', emailEntry: '', addedPhotos: [] });
```

Comment:
The reason I have that is because when a user types in text and submits their answer or question, unless they actively delete the text, it will still show up there, so it is sort of a temporary way of signalling that their answer has been submitted (I was thinking of nesting another modal if I can find time!).